Dockerfile.dev

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+ARG PYTHON_VERSION=3.11
```

Comment:
Add comment:

This image is depending on the builder image to serve as a development environment for either the Plone 'content api' backend for Volto or the Plone classicUI server